docs/spark/spark-configuration.md

```diff
@@ -185,5 +185,5 @@ df.write
 | snapshot-property._custom-key_    | null            | Adds an entry with custom-key and corresponding value in the snapshot summary  |
 | fanout-enabled       | false        | Overrides this table's write.spark.fanout.enabled  |
 | check-ordering       | true        | Checks if input schema and table schema are same  |
-| isolation-level | null | Desired isolation level for Dataframe overwrite operations.  Null => no checks, `serializable` => check for concurrent inserts or deletes in destination partitions, `snapshot` => checks for concurrent deletes in destination partitions. |
-| validate-from-snapshot-id | null | If isolation level is set, identifies base snapshot from which to check concurrent conflicts. If null, check from the first un-expired snapshot.  Obtain via `Spark3Util.loadIcebergTable().currentSnapshot().snapshotId()` |
+| isolation-level | null | Desired isolation level for Dataframe overwrite operations.  `null` => no checks (for idempotent writes), `serializable` => check for concurrent inserts or deletes in destination partitions, `snapshot` => checks for concurrent deletes in destination partitions. |
+| validate-from-snapshot-id | null | If isolation level is set, identity of base snapshot from which to check concurrent write conflicts into a table. Should be the snapshot before any reads from the table. Can be obtained via [Table API](../../api#table-metadata) or [Snapshots table](../spark-queries#snapshots). If null, checks from the table's first un-expired snapshot. |
```

Comment:
I think it would be more clear if "identity of" were replaced with "id of". We often refer to snapshot IDs as simply "id"

Reply:
Also, "checks from" would be more clear as "If null, the table's oldest known snapshot is used"